src/domain/app/i18n/sv.json

```
@@ -506,6 +506,7 @@
           "paragraph3": "Anteckna även volontärens <a href=\"https://q.surveypal.com/thinksustainably-Tapahtumat-kriteerit-2021/0\" target=\"_blank\" rel=\"noopener noreferrer\" aria-label=\"ansvarsinformation {{openInNewTab}}\">ansvarsinformation</a>. Att fylla i formuläret tar bara cirka 2 minuter och det öppnas i ett nytt fönster. Du kommer inte att förlora informationen i detta formulär."
         }
       },
+      "infoTextEnvironmentalCertificate": "Om företaget eller evenemanget har ett certifikat är dess namn obligatorisk information (t.ex. ekokompass). Om det inte finns något certifikat visar certifikatfältet ingenting.",
```

```suggestion
      "infoTextEnvironmentalCertificate": "Om företaget eller evenemanget har ett certifikat är dess namn obligatorisk information (t.ex. ekokompass). Om det inte finns något certifikat lämnas certifikatfältet tomt.",
```
Suggestion for the same reason as for the English translation.